lib/rubocop/cop/rspec/repeated_example_group_body.rb

```diff
@@ -43,6 +52,7 @@ class RepeatedExampleGroupBody < Cop
 
         def_node_matcher :metadata, '(block (send _ _ _ $...) ...)'
         def_node_matcher :body, '(block _ args $...)'
+        def_node_matcher :const_arg, '(block (send _ _ $(const _ _) ...) ...)'
```

Comment:
yeah, i don't entirely understand how node patterns work, this was mostly created with trying stuff until it got what i wanted

Reply:
I tried, i don't think it's possible, i don't think even combining the metadata and body matchers is possible because of the possibility of groups with no arg.

Reply:
> I don't entirely understand how node patterns work, this was mostly created with trying stuff until it got what i wanted

I struggle with it every time as well, tried to remember how named captures work and ended up looking up in the documentation yesterday when playing with your pull request :D

> don't think even combining the metadata and body matchers is possible because of the possibility of groups with no arg.

You're right, I haven't thought about this 👍